src/config.c

```diff
@@ -3247,5 +3254,11 @@ const command_rec oidc_config_cmds[] = {
 				RSRC_CONF|ACCESS_CONF|OR_AUTHCFG,
 				"Define the cookie prefix for the state cookie."),
 
+		AP_INIT_TAKE1(OIDCCABundlePath,
+				oidc_set_string_slot,
```

Comment:
@spanglerco suggested calling `ap_server_root_relative` so that relative paths will work as well. I'll update with that change.